chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -506,24 +505,33 @@ private void putCertResultActivityInRow(final QuestionableActivityCertificationR
         currRow.set(VERSION_COL, activity.getListing().getVersion().getVersion());
         currRow.set(LISTING_COL, activity.getListing().getChplProductNumber());
         currRow.set(STATUS_COL, activity.getListing().getCertificationStatusName());
-        currRow.set(LINK_COL, env.getProperty("chplUrlBegin") + env.getProperty("listingReportsUrlPart") + "/"
-                + activity.getListing().getId());
-        currRow.set(ACTIVITY_USER_COL, activity.getUser().getUsername());
+        currRow.set(LINK_COL, env.getProperty("chplUrlBegin") + env.getProperty("listingReportsUrlPart") + "/" + activity.getListing().getId());
+        currRow.set(ACTIVITY_USER_COL, activity.getUser().getSubjectName());
 
         String currActivityRowValue = currRow.get(ACTIVITY_DESCRIPTION_COL);
         if (!StringUtils.isEmpty(currActivityRowValue)) {
             currActivityRowValue += "; ";
         }
 
         if (activity.getTrigger().getName().equals(QuestionableActivityTriggerConcept.G1_SUCCESS_EDITED.getName())
+                || activity.getTrigger().getName().equals(QuestionableActivityTriggerConcept.G2_SUCCESS_EDITED.getName())) {
```

Comment:
Looking at the resulting file, I think I see three places where the output of all of these if/else statements is the same `+=`. In those places it also looks like there's some duplication of `if` clauses. For instance, the check against GAP_EDITED could be true in both `if`s on lines 525 and 527.

There are two places where I think we should consider combining things; the `G1/G2 MEASURE ADDED` and `REPLACED SVAP ADDED` results in almost the same thing, and given that SVAP hasn't been put in the system yet, I suggest we make the output of the SVAP addition QA mimic the format of the addition of the G1/G2 QA stuff

Also, can we replace this whole thing with a `switch` statement? It's really hard to read as is